getting-started/CONCEPTS/PLANNER.md

```diff
@@ -0,0 +1,85 @@
+# Planner
+
+The planner is takes in the user's ask and returns back a plan on how to accomplish the request. The user's ask is in the form of a prompt or prompt template. It does this by using AI to mix and match atomic functions (called _actions_) registered to the AI system so that it can recombine them into series of steps that complete a goal.
```

Comment:
```suggestion
The planner receives the user's ask and returns a plan on how to accomplish the request. The user's ask is in the form of a prompt or prompt template. It does this by using AI to mix and match atomic functions (called _actions_) registered to the AI system so that it can recombine them into a series of steps that complete a goal.
```